frontend/src/components/auth/Logout.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {logout} from '../../actions/authActions';
+import { Menu } from 'antd';
+import { Redirect} from 'react-router-dom';
+class Logout extends React.Component {
+    render() {
+        
+        let {logout: logouter, ...rest} = this.props;
```

Comment:
Is this a good way to update the attributes in state?

I thought there should have some other ways to check.

Reply:
yeah, because the state is app wide, and all relevant components are checking on render for authentication, as soon as logout button is clicked and state is cleared, no matter what component we are at, we will be redirected to the landing page

Reply:
Sorry, Let me rephrase. There should have other ways to update the attributes in state. Directly pushing props into the state in component is an approach I have never seen before. 

If you feel good about this then ignore this msg. But I would suggest to have functions inside components for state updates.